interface/src/app/analyze/experiment/experiment.js

```diff
@@ -9,11 +9,7 @@ angular.module('adage.analyze.experiment', [
     '/api/v0/experiment/:accession/',
     // TODO need to add logic for handling pagination of results.
     // then, can change "limit" below to something sensible
-    {accession: '@accession', limit: 0},
-    // Angular expects a query service to give only a list but our Tastypie
-    // interface wraps the response list with pagination so we tell Angular to
-    // expect an object instead via isArray: false
-    {'get': {method: 'GET', isArray: false}}
+    {limit: 0}
```

Comment:
Nice simplification @mhuyck and @dongbohu! I guess this will still allow us to request an experiment by its accession number, right?